content/en/blog/2024/go-contrib-removal.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Abandoned Go Contrib modules need code owners or will be removed
+linkTitle: Go Contrib modules ownership
+date: 2024-05-27
+author: >-
+  [Fabrizio Ferri-Benedetti](https://github.com/theletterf) (Splunk)
+issue: 4542
+sig: SIG Go
+# prettier-ignore
+cSpell:ignore: aws Benedetti Fabrizio Ferri gopkg labstack macaron moduled otelaws otelecho otellambda otelmacaron otelmongo otelmux
+---
+
+The Go SIG will remove Contrib modules that lack code owners. For each module,
+the removal is planned for no sooner than August 21, 2024, unless a code owner
+is found. The list includes the following modules:
```

Comment:
Two comments here:

- Maybe clarify that such modules are considered deprecated as of the date of this post
  - If this is the case, will they all be marked as deprecated by the SIG, e.g. https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5636?
- August 21 seems like an arbitrary day. Could it be EOM or maybe Sept 1? (See the suggestion below.)

Also, maybe this could be rephrased to read something like: 

```suggestion
The Go SIG will remove contrib modules that lack code owners starting September 1, 2024.

Currently unowned modules include the following:
```

Reply:
@pellared @dmathieu WDYT about the the above questions?

Reply:
> If this is the case, will they all be marked as deprecated by the SIG,

Yes.

> August 21 seems like an arbitrary day

Let's please keep this date as we already have this date in a lot of places and change it will be a PITA 😉